netcdf/src/error.rs

```diff
@@ -129,7 +129,7 @@ use std::fmt;
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            Self::Str(x) => write!(f, "{}", x),
+            Self::Str(x) => write!(f, "{x}"),
```

Comment:
```suggestion
            Self::Str(x) => x.fmt(f),
```

Reply:
I find this awkward and inconsistent with the other entries, is there a benefit to this in terms of code generation?

Reply:
Yeah, it produces much less code, at least until someone changes it back to add an error message 😅

Reply:
Makes sense, I changed it to match your suggestions